src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -1,46 +1,51 @@
 import React from 'react';
 import Button from '../Button/Button';
 
-const buttonPanel = () => {
+const buttonPanel = (props) => {
+
+
+  const handleClick = (buttonName) => {
```

Comment:
## [IMPORTANT CHANGE]

`ButtonPanel` is simply a "bridge" between `App.js` and `Button` for the `clickHandler`. This isn't the place where we provide the `btnNames` . We could have simply passed `clickHandler` directly onto the `Button` render:

```javascript
// one of the Button renders:
<Button name = "AC" color = "#E0E0E0" clickHandler = {clickHandler}/>
// assuming props was deconstructed that's why i didn't write it as props.clickHandler
```

Reply:
I think there has been an update since the last time... I basically followed the steps outlined in the Microverse pathwright to create this. 

I was required to group the buttons in the button panel into `group1..group5`

Reply:
Im not sure what you are saying and how it's related to the comment I made :) Can you be more clear?

Reply:
I mean I could have done it any other way but the Microverse pathwright instruction for this project outlined a particular series of steps and that's what I followed. It stated explicitly that I should have an `handleClick` function in my `buttonPanel.js` which takes in buttonName as an argument and returns `clickHandler` from props. 

[Please check this out](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896095/)

Reply:
Ok I got what you mean. Anyway, we're in the process of improving the instructions. So consider the comment I made as a step towards that end as well. I mean, my comment and change request stands :)

Reply:
Oh okay then. I will implement it right away :slightly_smiling_face:Thanks